includes/blocks/course-theme/class-next-lesson.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * File containing the Next_Lesson class.
+ *
+ * @package sensei
+ * @since
```

Comment:
Yep. The version number here is empty because I didn't know which version release will be a Course Theme release version. Do you want me to use a particular version here? Or do you want me to put a placeholder so we can swap it when the version is decided? For example
```
/**
 * @since course_theme_since_version
 */
```

Reply:
Yeah! Currently, this is a problem. But here we can add the next release that is the [3.13.4](https://github.com/Automattic/sensei/milestones), for now (if we release before the merge, we need to replace it in our open PRs). The project will be launched probably in `3.16.0` or `4.0`, but we'll already release parts of the code behind the feature flag.

In the future, we can think of marks, and replace them with our release scripts. :)